### ISSUE TITLE
DynamoDbLocal breaks my logs

### ISSUE THREAD

**Issue by furstenheim-goodnotes** [awslabs/amazon-dynamodb-local-samples#8]
I'm installing `com.amazonaws:DynamoDBLocal:2.0.0+` in gradle with `testImplementation(com.amazonaws:DynamoDBLocal:2.0.0+)`.

Doing so I get the following warn:

```
SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
SLF4J: Class path contains SLF4J bindings targeting slf4j-api versions 1.7.x or earlier.
SLF4J: Ignoring binding found at [jar:file:myhome/repository/ch/qos/logback/logback-classic/1.2.11/logback-classic-1.2.11.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See https://www.slf4j.org/codes.html#ignoredBindings for an explanation.
```

And all my app logs disappear. Just commenting the import makes the logs work again.

I'd be grateful for any help.

Cheers

**Comment by furstenheim-goodnotes** [comment]
Excluding a conflicting org.eclipse.jetty fixed it
```
testImplementation("com.amazonaws:DynamoDBLocal:2.0.0+") {
   exclude(group = "org.eclipse.jetty")
} 
```

**Comment by shetsa-amzn** [comment]
We appreciate your feedback and we are currently reviewing this. We will update as soon as we have more information about next steps.

**Comment by shetsa-amzn** [comment]
Today we have released version 2.3.0 for DynamoDBLocal, this contains upgrades to JDK from 11 to 17 and Jetty from 11 to 12. Can you please check the issue still persist with latest updates. 

https://aws.amazon.com/about-aws/whats-new/2024/03/amazon-dynamodb-local-upgrades-jetty-12-jdk-17/

Thanks
